Formula/openjdk.rb

```diff
@@ -2,19 +2,20 @@ class Openjdk < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.java.net/"
   if Hardware::CPU.arm?
-    # Temporarily use a openjdk 16 preview on Apple Silicon
+    # Temporarily use a openjdk 17 preview on Apple Silicon
     # (because it is better than nothing)
-    url "https://github.com/openjdk/jdk-sandbox/archive/a56ddad05cf1808342aeff1b1cd2b0568a6cdc3a.tar.gz"
-    sha256 "29df31b5eefb5a6c016f50b2518ca29e8e61e3cfc676ed403214e1f13a78efd5"
-    version "15.0.2"
+    url "https://github.com/openjdk/jdk/archive/refs/tags/jdk-17+22.tar.gz"
+    sha256 "e1ad749a8b689102a05fbb222d2560e6cc49f675df1715b921be5e3f7660c11b"
+    version "17"
```

Comment:
This `version` combined with `livecheck` is causing audit failure for ARM due to trying to access non-existent repo.

Not sure if it is better to have a fake version number (and potentially other `version` logic like previously) or use a dedicated `livecheck`.

Reply:
We need to put the same version here as the one used on Intel. Mixing the version across arches doesn't work. See also `go` when we shipped it as 1.15 on Intel but 1.16 on ARM.